sources/niLang/src/API/niLang/IDataTable.h

```diff
@@ -168,7 +168,10 @@ struct iDataTable : public iUnknown
   virtual tU32 __stdcall GetNumChildren() const = 0;
   //! Get the index of the first child table with the specified name.
   //! {Property}
-  virtual tU32 __stdcall GetChildIndex(const achar* aaszName) const = 0;
+  virtual tU32 __stdcall GetChildIndex(const achar *aaszName) const = 0;
+  //! Get the index of the child table.
+  //! {Property}
+  virtual tU32 __stdcall GetChildIndexFromDataTable(iDataTable *apDT) const = 0;
```

Comment:
I think this can be removed since we already have `iDataTable::GetIndex()` which return the index in its parent.